apps/qubit/modules/actor/templates/browseSuccess.php

```diff
@@ -68,14 +68,14 @@
         <?php echo get_partial('search/aggregation', array(
           'id' => '#facet-places',
           'label' => sfConfig::get('app_ui_label_place'),
-          'name' => 'places',
+          'name' => 'place',
```

Comment:
Why is this changed?

Reply:
This facet isn't currently working in qa/2.5.x so I think I misnamed it at some point. Same with subject.